resources/resinos.robot

```diff
@@ -119,7 +119,7 @@ Check if kernel module loading works on "${device_uuid}"
     [Teardown]    Run Keyword    "disable" public URL for device "${device_uuid}"
 
 Run "${image}" on "${application_name}" with delta already enabled
-    ${random} =   Evaluate    random.randint(0, sys.maxint)    modules=random, sys 
+    ${random} =   Evaluate    random.randint(0, sys.maxint)    modules=random, sys
```

Comment:
@horia-delicoti random line change?

Reply:
@ZubairLK just an empty space at the end of the line.